src/test/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/integration/GoalControllerTest.kt

```diff
@@ -616,7 +623,7 @@ class GoalControllerTest : IntegrationTestBase() {
     @Transactional
     open fun `remove goal without a note should fail`() {
       val goalUuid = "31d7e986-4078-4f5c-af1d-115f9ba3722d"
-      val note = Goal(note = "")
+      val note = Goal(note = "", reminderDate = LocalDate.now().plusWeeks(20).toString())
```

Comment:
This reminderDate value is never asserted. Should it be?

Reply:
I have removed reminderDate completely from here, it is not needed.